scripts/descriptors/messagebottlemanager.lua

```diff
@@ -0,0 +1,21 @@
+local function Describe(self, context)
+	local count = 0
+
+	for key, value in pairs(self.active_treasure_hunt_markers) do
+		count = count + 1
+	end
+
+	local description = string.format(context.lstr.messagebottlemanager, count, 10)
```

Comment:
This should be TUNING.MAX_ACTIVE_TREASURE_HUNTS

Reply:
Fixed